examples/client/Locomotion/src/pages/Payments/cardDetailUtils.ts

```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (string: string) => string?.charAt(0).toUpperCase() + string?.slice(1);
+export const getLastFourForamttedShort = (lastFour: string) => `**** ${capitalizeFirstLetter(lastFour)}`;
+export const getLastFourForamttedShortLong = (lastFour: string) => `**** **** **** ${capitalizeFirstLetter(lastFour)}`;
```

Comment:
remove Short

Reply:
in choose payment method and credit card list we use short